Resources/Server Info/Guidebook/Chef/salad.xml

```diff
@@ -0,0 +1,151 @@
+<Document>
+
+# Salads
+
+- [color=#fcdf03]All salads need a[/color] [color=#a4885c]large bowl[/color] [color=#fcdf03]along with everything else![/color]
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Herb Salad[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- Bowl
+- 3x Ambrosia Vulgaris
+- Apple
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="5 sec"/>
+<GuideEntityEmbed Entity="FoodSaladHerb" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Coleslaw[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- 5u vinaigrette
+- Bowl
+</Box>
+<Box Orientation="Vertical">
+- Red onion
+- Cabbage
+</Box>
+<GuideEntityEmbed Entity="KitchenMicrowave" Caption="5 sec"/>
+<GuideEntityEmbed Entity="FoodSaladColeslaw" Caption=""/>
+</Box>
+
+<Box Orientation="Vertical">
+	[color=#a4885c]Caesar Salad[/color]
+</Box>
+<Box>
+<Box Orientation="Vertical">
+- 5u olive oil
+- Bowl
+- Slice of red onion
+</Box>
+<Box Orientation="Vertical">
+- Slice of bread
+- Cheese wedge
```

Comment:
Wouldn't mozzarella fit Caesar salad more?

Reply:
Caesar salad was made for upstream just so I could get my cabbage so why not if it makes more sense